.doctrine-project.json

```
@@ -0,0 +1,48 @@
+{
+  "active": true,
+  "name": "Doctrine Module for Laminas",
+  "shortName": "Doctrine for Laminas",
+  "slug": "DoctrineModule",
+  "docsSlug": "doctrine-module",
+  "versions": [
+    {
+      "name": "4.0.x",
+      "branchName": "4.0.x",
+      "slug": "4.0.x",
+      "current": true,
+      "maintained": true,
+      "aliases": [
+        "current",
+        "stable"
+      ]
+    },
+    {
+      "name": "4.1.x",
+      "branchName": "4.1.x",
+      "slug": "4.1.x",
+      "current": false,
+      "maintained": true,
+      "aliases": [
+        "future"
```

```suggestion
        "latest"
```
would be consistent with the rest of the projects.

---

But 4.1.x is not the latest; there have been releases of 4.0.x since.  I would be comfortable adding "latest" to 4.0.x and not using any aliases for 4.1.x  Would that be ok?

---

4.0.x is stable and 4.1.x is not so using latest seems ok for 4.1.x